### PR TITLE
fix: cleanup of all updatecli manifests

### PR DESCRIPTION
**Pull request by lemeurherve**
- Replaced deprecated main `title` by `name`
- Added missing `name` field in all `sources` blocks
- Replaced deprecated `depends_on` by `dependson`
- Replaced deprecated and now useless `targets` by `title` in all `pullrequest` blocks
- Completed and fixed labels in all `pullrequests` blocks

### CODE REVIEW

**Review by dduportal — APPROVED** [review]
❤️

**Review by dduportal — CHANGES_REQUESTED** [review]
The checks have 2 errors:

```
[2022-09-23T10:32:03.084Z] ERROR: failed loading pipeline(s)
[2022-09-23T10:32:03.084Z] 	* "updatecli/updatecli.d/charts/nginx-ingress.yaml" - yaml: unmarshal errors:
[2022-09-23T10:32:03.084Z]   line 19: mapping key "name" already defined at line 17
```

and 

```
[2022-09-23T10:32:22.974Z] ######################################
[2022-09-23T10:32:22.974Z] # BUMP `DATADOG` DOCKER IMAGE DIGEST #
[2022-09-23T10:32:22.974Z] ######################################
[2022-09-23T10:32:22.974Z] 
[2022-09-23T10:32:22.974Z] 
[2022-09-23T10:32:22.974Z] SOURCES
[2022-09-23T10:32:22.974Z] =======
[2022-09-23T10:32:22.974Z] Pipeline "Bump `datadog` docker image digest" failed
[2022-09-23T10:32:22.974Z] Skipping due to:
[2022-09-23T10:32:22.974Z] 	sources stage:	"template: cfg:5:58: executing \"cfg\" at <source \"latestRelease\">: error calling source: key not defined in configuration"
```

File: updatecli/updatecli.d/charts/nginx-ingress.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ sources:
   lastChartVersion:
     name: "Retrieve latest version of the chart ingress-nginx"
     kind: helmchart
+    name: get last chart version
```
Comment (dduportal) [inline updatecli/updatecli.d/charts/nginx-ingress.yaml:19]:
```suggestion
```

Duplicates thes line 17

File: updatecli/updatecli.d/docker-images/datadog.yaml [hunk 2]
```diff
@@ -38,8 +38,7 @@ pullrequests:
   default:
     kind: github
     scmid: default
-    targets:
-      - updateDigestInConfig
+    title: Bump `datadog`` docker image version to {{ source "latestRelease" }}
```
Comment (lemeurherve) [inline updatecli/updatecli.d/docker-images/datadog.yaml:41]:
```suggestion
    title: Bump `datadog` docker image version to {{ source "latestDigest" }}
```

[the remark this answers is not in the document]
Comment (lemeurherve) [reply updatecli/updatecli.d/docker-images/datadog.yaml]: There was also an issue with the doubled backtick, fixed in another suggestion.

**Review by dduportal — APPROVED** [review]

**Pull request merged**